playbook-examples/primary_tde_only/inventory.yml

```diff
@@ -0,0 +1,9 @@
+---
+all:
+  children:
+    primary:
+      hosts:
+        primary1:
+          ansible_host: xxx.xxx.xxx.xxx
+          private_ip: xxxx.xxxx.xxx.xxx
+          hostname: primary
```

Comment:
remove this line

Reply:
Why?

Reply:
IIRC, the intention was to showcase the ability to override the hostname by setting this variable. However, it looks like we rolled back that capability somewhere.